evap/evaluation/migrations/0116_remove_contributon_can_edit.py

```diff
@@ -0,0 +1,15 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('evaluation', '0115_add_contribution_role'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
could be added to the previous migration but like this it's fine for me as well

Reply:
Sadly, no. I get an error like this when trying to put everything in a single migration:
```
django.db.utils.OperationalError: cannot ALTER TABLE "evaluation_contribution" because it has pending trigger events